packages/platform-browser/src/Permissions.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Wrapper on the Permission API (`navigator.permissions`)
+ * with methods for querying status of permissions.
+ *
+ * @since 1.0.0
+ */
+
+import type { PlatformError } from "@effect/platform/Error"
+import type { Tag } from "effect/Context"
+import type * as Effect from "effect/Effect"
+import type * as Layer from "effect/Layer"
+import * as internal from "./internal/permissions.js"
+
+/**
+ * @since 1.0.0
+ * @category interface
+ */
+export interface Permissions {
```

Comment:
This will need a `TypeId` to prevent it from being merged.

Reply:
Should we add it also for `Clipboard`? I can open another PR for that

https://github.com/Effect-TS/effect/blob/5392047b3398cb4ae82e87ec1d77279c23661e8a/packages/platform-browser/src/Clipboard.ts#L15-L22

Reply:
Yes, if you want to PR the change go for it.